ember-container-query/package.json

```diff
@@ -58,22 +58,23 @@
     "@babel/plugin-proposal-decorators": "^7.20.7",
     "@babel/preset-typescript": "^7.18.6",
     "@embroider/addon-dev": "^3.0.0",
-    "@gavant/glint-template-types": "^0.3.0",
+    "@gavant/glint-template-types": "^0.3.1",
     "@glint/core": "^v1.0.0-beta.2",
     "@glint/environment-ember-loose": "^v1.0.0-beta.2",
     "@glint/template": "^v1.0.0-beta.2",
     "@tsconfig/ember": "^2.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.48.1",
-    "@typescript-eslint/parser": "^5.48.1",
+    "@typescript-eslint/eslint-plugin": "^5.48.2",
+    "@typescript-eslint/parser": "^5.48.2",
     "ember-template-lint": "^5.3.1",
     "ember-template-lint-plugin-prettier": "^4.1.0",
-    "eslint": "^8.31.0",
+    "eslint": "^8.32.0",
     "eslint-config-prettier": "^8.6.0",
-    "eslint-plugin-ember": "^11.4.2",
-    "eslint-plugin-n": "^15.6.0",
+    "eslint-plugin-ember": "^11.4.3",
+    "eslint-plugin-n": "^15.6.1",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-simple-import-sort": "^8.0.0",
+    "eslint-plugin-simple-import-sort": "^9.0.0",
     "eslint-plugin-typescript-sort-keys": "^2.1.0",
+    "prettier": "^2.8.3",
```

Comment:
I forgot to add `prettier` earlier. (See https://github.com/ijlee2/ember-container-query/pull/151/commits/1ac7a4d29b4a118fc1e9fedae3d3646f14c0faca#r1072256280)